semgrep-core/src/analyzing/Dataflow_svalue.ml

```diff
@@ -383,7 +389,10 @@ let rec is_symbolic_expr expr =
   | G.ArrayAccess (e1, (_, e2, _)) -> is_symbolic_expr e1 && is_symbolic_expr e2
   | G.Call (e, (_, args, _)) ->
       is_symbolic_expr e && List.for_all is_symbolic_arg args
-  | _ -> false
+  | G.New (_, _, args) ->
+      let args = G.unbracket args in
+      List.for_all is_symbolic_arg args
+  | _else -> false
```

Comment:
Nit: Is `_else` a new standard we're trying to adopt? I find `_` perfectly clear; if it's not I can start using `_else` but we should standardize

Reply:
I find `| _ -> ...` perfectly clear as I would find ` | $%! -> ...` perfectly clear if `$%!` was the OCaml syntax for the wildcard pattern. I adopted this practice from Haskell land time ago (the GHC project). If people feel confused about it we can vote to forbid anything than `| _ -> ...`. For me `_else` is easier to read at least.